cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -81,6 +76,12 @@ def test_vm_mac_scvmm():
         caseimportance: low
         initialEstimate: 1/20h
         setup: https://bugzilla.redhat.com/show_bug.cgi?id=1514461
+        testSteps:
+            1. Add SCVMM to CFME
```

Comment:
It would be better to write description or steps for ```setup``` rather than providing BZ link. Although you have already mentioned BZ above.

Reply:
Actually this `setup` is unnecessary, as it's covered by the `testSteps`.